docs/api-doc.md

```diff
@@ -1,5 +1,5 @@
 
-# dk-api
+# dk-api Copy 2
```

Comment:
what is 'Copy' for?

Reply:
Forgot to change the collection name when generating the doc lol. Will fix n push